doc_src/string.txt

```diff
@@ -106,6 +106,7 @@ The following subcommands are available:
 <bs>c</bs>g
 \endfish
 
+<<<<<<< c1ddfa5c636b8b353e56e10b66601ad1751cafcc
```

Comment:
Geez, didn't notice that. Will have an invert-match-1 tag with that resolved soon.